Example/Example.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this?

Reply:
I think we can get rid of the entire workspace, to be honest. I'll add it to gitignore.